src/GameEngine/Logik.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameEngine
```

Comment:
I would normally prefer that you do one file per class, so one file called Player.cs containing the Player class